Gen3_Source_injection_Calexp.ipynb

```diff
@@ -0,0 +1,560 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    def is_in_region(ra, 
</pre><p>this function is also defined in another notebook. I guess these functions can be defined in a package somewhere (either in the SL pipeline stack or somewhere else on the LSST pipeline stack) such that they can be re-used. I understand that you first code them up in this notebook. Would be better for prosperity and to avoid duplications</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/DC2-notebooks/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>